src/main/java/net/neoforged/neoforge/server/ServerLifecycleHooks.java

```diff
@@ -135,6 +145,48 @@ public static void handleExit(int retVal) {
         System.exit(retVal);
     }
 
+    private static final ArrayList<SchemaComponent> SCHEMA_REGISTRY_VIEW = new ArrayList<>();
+
+    @Internal
+    public static List<SchemaComponent> getSchemaRegistry() {
+        return SCHEMA_REGISTRY_VIEW;
+    }
+
+    @Internal
+    public static void fireSchemaRegistryEvent() {
+        final Map<String, SchemaComponent> schemaRegistry = new ConcurrentSkipListMap<>(String::compareTo);
```

Comment:
```suggestion
        final Map<String, SchemaComponent> schemaRegistry = new HashMap<>();
```
and sort the final list below